bootstrap/bootstrap.py

```diff
@@ -378,7 +384,11 @@ def main() -> None:
                         help='path to prefetched bootstrap sources archive')
     parser.add_argument('--archive', dest='want_archive', action='store_true')
     parser.add_argument('--no-archive', dest='want_archive', action='store_false')
+    parser.add_argument('-j', type=int, metavar="NUM",
+                        help='specify the number of jobs (commands) to run simultaneously')
+
     parser.set_defaults(want_archive=True)
+    parser.set_defaults(j=1)
```

Comment:
I'd prefer that every `set_default` followed its respective `add_argument` and these groups were split with empty lines. I.e.

```suggestion
    parser.set_defaults(want_archive=True)

    parser.add_argument('-j', type=int, metavar="NUM",
                        help='specify the number of jobs (commands) to run simultaneously')
    parser.set_defaults(j=1)
```

Reply:
Done